sirepo/package_data/static/js/raydata.js

```diff
@@ -677,13 +682,27 @@ SIREPO.app.directive('scansTable', function() {
             }
 
             $scope.arrowClass = column => {
-                if (scanArgs.sortColumn !== column) {
-                    return {};
-                }
-                return {
-                    glyphicon: true,
-                    [`glyphicon-arrow-${scanArgs.sortOrder ? 'up' : 'down'}`]: true,
-                };
+                let r = {};
+                scanArgs.sortColumns.forEach((sortField) => {
+                    if (sortField[0] === column) {
+                        r= {
+                            glyphicon: true,
+                            [`glyphicon-arrow-${sortField[1] ? 'up' : 'down'}`]: true,
+                        };
+                    }
+                });
+                return r;
+            };
+
+            $scope.arrowStyle = column => {
+                let r = '';
+                scanArgs.sortColumns.forEach((sortField, i) => {
+                    if (sortField[0] === column) {
+                        let a = i === 0 ? 1 : (-0.3 * i) + 0.8;
```

Comment:
i === 0 deserves a comment. I think it is because sortColumns is ordered from highest to lowest priority but it would be good to let the reader know that.

I think we might want an assert that sortColumns.length <= 3. `(-0.3 * i) + 0.8` goes negative at index 3 and above which I don't think is valid. Not 100% about this.

Reply:
if `a` goes negative, the arrow just won't show, so the behavior is the same. [7322d19](https://github.com/radiasoft/sirepo/pull/7163/commits/7322d198d8db767467b2dfee55a74b6b9b0cba27)